apps/package.json

```diff
@@ -269,6 +195,7 @@
     "@dsco_/link": "^1.1.2",
     "@lezer/highlight": "^1.2.0",
     "@microsoft/immersive-reader-sdk": "^1.1.0",
+    "@react-bootstrap/pagination": "^1.0.0",
     "@reduxjs/toolkit": "^1.9.3",
     "@types/chai": "^4.3.5",
```

Comment:
Since Chai, Mocha, and Sinon are all used for testing, should the @types/chai, etc. all be up in devDependencies?

Reply:
That makes sense to me, I'll move them and see if anything freaks out :)

Reply:
seemed to work ok, fixed!